tensorflow/docs_src/programmers_guide/tensors.md

```diff
@@ -197,7 +197,7 @@ For example, here is how to make a vector of zeros with the same size as the
 number of columns in a given matrix:
 
 ``` python
-zeros = tf.zeros(tf.shape(my_matrix)[1])
+zeros = tf.zeros(sess.run(tf.shape(my_matrix)[1]))
```

Comment:
How about using my_matrix.shape[1] instead. This is more efficient than going though sess.run()

Reply:
done!